prijateli_tree/app/routers/games.py

```diff
@@ -198,6 +206,22 @@ def get_previous_answers(
     }
 
 
+def language_check(player_lang, languages):
+    """
+    Helper function to ensure that the language is in the languages dict
+    """
+    if player_lang not in languages:
+        player_lang = player_lang.lower()
```

Comment:
The language keys are already lowercase, this shouldn't be necessary.

Reply:
Ok! I did this because they were uppercase in my env, so probably just need to update

Reply:
It was updated here: https://github.com/prijatelilab/PrijateliTree/pull/51/files#r1398307156